lib/src/traceable_widget_mixin.dart

```diff
@@ -4,29 +4,33 @@ import 'package:matomo_tracker/matomo_tracker.dart';
 /// Register a `trackScreenWithName` on this widget.
 @optionalTypeArgs
 mixin TraceableClientMixin<T extends StatefulWidget> on State<T> {
-  /// {@template traceableClientMixin.traceName}
+  /// {@template traceableClientMixin.actionName}
   /// Equivalent to an event action. (e.g. `'Created HomePage'`).
-  ///
-  /// This corresponds with `action_name`.
   /// {@endtemplate}
   @protected
-  String get traceName => 'Created widget ${widget.toStringShort()}';
+  String get actionName => 'Created widget ${widget.toStringShort()}';
 
-  /// {@template traceableClientMixin.traceTitle}
+  /// {@template traceableClientMixin.eventName}
   /// Equivalent to an event name. (e.g. `'HomePage'`).
   ///
   /// This corresponds with `e_n`.
   /// {@endtemplate}
   @protected
-  String get traceTitle;
+  String get eventName;
```

Comment:
The mixin should track a page view and not an event; there should not be an eventName here.